src/ucm/bistro/bistro.c

```diff
@@ -50,12 +50,16 @@ static ucs_status_t ucm_bistro_protect(void *addr, size_t len, int prot)
 
 void ucm_bistro_modify_code(void *dst, const ucm_bistro_lock_t *bytes)
 {
-    uint32_t value;
-
-    UCS_STATIC_ASSERT(sizeof(*bytes) <= sizeof(value));
-    memcpy(&value, dst, sizeof(value));
-    memcpy(&value, bytes, sizeof(*bytes));
-    (void)ucs_atomic_swap32(dst, value);
+    UCS_STATIC_ASSERT(sizeof(*bytes) == 2 || sizeof(*bytes) == 4);
```

Comment:
although not strictly needed, as per coding style, please add parentheses like `ASSERT((sizeof...) || (sizeof...))`, and blank line after.

Reply:
Updated, thank you!